server/server.js

```diff
@@ -53,6 +53,11 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         res.sendStatus(200);
     });
 
+    app.get("/admin/logout", function(req, res) {
```

Comment:
I don't think this should be a GET endpoint; GET is supposed to always be idempotent and cacheable, which logging out isn't